scrapers.js

```diff
@@ -1486,7 +1486,33 @@ let scrapers = [
       })
       return counties;
     }
-  }
+  },
+  {
+    state: 'MD',
+    country: 'USA',
+    url: 'https://coronavirus.maryland.gov/',
+    scraper: async function() {
+      let counties = [];
+      let $ = await fetch.headless(this.url);
+      let paragraph = $('p:contains("Number of Confirmed Cases:")').next('p').text();
+  
+      paragraph.split(')').map(splitCounty => {
+      if(splitCounty.length > 1){
+       let county = parse.string(splitCounty.substring(0, splitCounty.indexOf('(')).trim())
+       //check for Baltimore City
+       if (county !== 'Baltimore City') {
+         county = transform.addCounty(county)
+       }
+       let cases = parse.number(splitCounty.substring(splitCounty.indexOf('(')+1, splitCounty.length).trim())
+       counties.push({
+         county,
+         cases
+       })}
+       
+      })
+      return counties;
```

Comment:
Please add this, it will sum all of the counties and consider it the state total:
```suggestion
      counties.push(transform.sumData(counties));

      return counties;
```

We need to do this for all states we have 100% complete data for, that way we override JHU with our own, known-good data.